horreum-web/src/components/SavedTabs.tsx

```diff
@@ -101,7 +101,7 @@ export default function SavedTabs(props: SavedTabsProps) {
                             const childProps = children[activeKey.current].props
                             if ("onSave" in childProps) {
                                 childProps
-                                    .onSave()
+                                    .onSave?.()
```

Comment:
I don't think this is actually required as `onSave` is a mandatory property, isn't it?
```suggestion
                                    .onSave()
```

Reply:
It has been a while since this change was made, but... iirc, i **think** I changed `SavedTabs.tsx` for the "Save" button to be optional, or at least that was the intention.  Not all tabs had a natural state that should be saved when navigating away from them, or I had moved some tabs that handling saving state slightly differently.

Reply:
yeah makes sense, I checked further and the props could be either `FragmentTabProps` or `SavedTabProps` and the `onSave` method is present only in the latter (where it must be always present). 
Moreover you are checking a couple of lines before if `onSave` is present in the child props so I think it is save to remove that `?.` notation. Anyway just a minor :)

Reply:
sure, change it to `.onSave()`